src/imitation/rewards/reward_nets.py

```diff
@@ -188,7 +210,7 @@ def device(self) -> th.device:
         return first_param.device
 
 
-class RewardNetShaped(RewardNet):
+class RewardNetShaped(AIRLRewardNet):
```

Comment:
Agreed that the naming is very confusing after my change. If we decide to go with the additional changes I'm proposing, there would only be one AIRL-specific class left, which would be called `AIRLRewardNet` (or similar), otherwise I'll rename the existing classes like you suggest. I'll hold off on that though until we've decided whether to implement my suggestions